README.md

```diff
@@ -47,48 +184,11 @@ To build a specific branch, you will need to use Maturin and have Rust installed
 maturin develop
 ```
 
-And you will be able to import bytewax in your python file.
-
-## Example
-
-Here is an example of a simple dataflow program using Bytewax:
-
-```python
-from bytewax.dataflow import Dataflow
-from bytewax.execution import run_main
-from bytewax.inputs import ManualInputConfig
-from bytewax.outputs import StdOutputConfig
-
-def input_builder(worker_index, worker_count, resume_from):
-    resume_from = None # Ignore recovery logic
-    for e in range(10):
-        yield resume_from, e
-
-flow = Dataflow()
-flow.input("input", ManualInputConfig(input_builder))
-flow.map(lambda x: x * x)
-flow.capture(StdOutputConfig())
-
-if __name__ == "__main__":
-    run_main(flow)
-```
-
-Running the program prints the following output:
+*Important*: If you are testing with a maturin built version from source, you should use `maturin build --release` since develop will be slower.
```

Comment:
```suggestion
*Important*: If you are testing with a maturin built version from source, you should use `maturin build --release` since `maturin develop` will be slower.
```